tests/lib/rules/computed-property-spacing.js

```diff
@@ -1040,7 +1040,7 @@ ruleTester.run("computed-property-spacing", rule, {
         {
             code: "class A { [a](){} get [b](){} set [c](foo){} static [d](){} static get [e](){} static set [f](bar){} }",
             output: "class A { [ a ](){} get [ b ](){} set [ c ](foo){} static [ d ](){} static get [ e ](){} static set [ f ](bar){} }",
-            options: ["always", {}],
```

Comment:
why removing this?

Reply:
same as https://github.com/eslint/eslint/pull/13034/files#r420341953
the with `{}` this is a valid case. 

perhaps I should add a valid case with same object. ?

this has become valid because there is no mutation is being done for an empty object `{}`